core/variant_call.cpp

```diff
@@ -1103,11 +1103,8 @@ struct _VariantCall {
 	}
 
 	static void Color_init3(Variant &r_ret, const Variant **p_args) {
-		r_ret = Color::html(*p_args[0]);
-	}
-
-	static void Color_init4(Variant &r_ret, const Variant **p_args) {
-		r_ret = Color::hex(*p_args[0]);
```

Comment:
Were these previous functions unused? :thinking:

Reply:
Not used since the "GODOT IS OPEN SOURCE" commit.